src/styleguide/_patterns/03-organisms/00-page-structure/02-hero.mustache

```diff
@@ -1,6 +1,6 @@
-<section class="grav-c-hero" style="background-image: url('{{hero.background}}')">
+<section class="grav-c-hero">
   {{#hero.content}}
```

Comment:
We should add the `<canvas>` element here (even though it's dormant without any CSS), as it's a useful reference to developers viewing the markup for the hero organism in our style guide.